pkg/sync/file/filepath_sync.go

```diff
@@ -46,20 +46,18 @@ func (fs *Sync) Sync(ctx context.Context, dataSync chan<- sync.DataSync) error {
 
 	dataSync <- sync.DataSync{FlagData: fetch, Source: fs.URI}
 
-	fs.Logger.Info(fmt.Sprintf("Watching filepath: %s", fs.URI))
+	fs.Logger.Info(fmt.Sprintf("watching filepath: %s", fs.URI))
 	for {
 		select {
 		case event, ok := <-watcher.Events:
 			if !ok {
-				fs.Logger.Info("Filepath notifier closed")
+				fs.Logger.Info("filepath notifier closed")
 				return errors.New("filepath notifier closed")
 			}
 
-			fs.Logger.Info(fmt.Sprintf("Filepath event: %s %s", event.Name, event.Op.String()))
+			fs.Logger.Info(fmt.Sprintf("filepath event: %s %s", event.Name, event.Op.String()))
 
 			switch event.Op {
-			case fsnotify.Create, fsnotify.Write:
```

Comment:
please revert this change

Reply:
https://github.com/open-feature/flagd/actions/runs/4032086735/jobs/6949951527

The code had been edited, likely during a merge but is resolved now